view/frontend/web/css/source/_module.less

```diff
@@ -12,3 +12,7 @@
  * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  */
 @import '_address-validation.less';
+
+.cart-totals tbody .totals-tax-details th {
```

Comment:
We also need to indent this on the checkout (not just the cart). Can you confirm that this code does that, and if not, add CSS to accomplish that?